mopidy_spotify/translator.py

```diff
@@ -315,7 +315,10 @@ def web_to_album(web_album):
     ]
     artists = [a for a in artists if a]
 
-    return models.Album(uri=ref.uri, name=ref.name, artists=artists)
+    # Note: date can by YYYY-MM-DD, YYYY-MM or YYYY.
+    date = web_album.get('release_date', '').split('-')[0] or None
+
+    return models.Album(uri=ref.uri, name=ref.name, artists=artists, date=date)
```

Comment:
I had to extend the track translator to parse release dates (just the bare minimum (year); spotify also supports finer dates) to support round-tripping between the web and native formats. otherwise, I wouldn't have been able to compare output from save() and lookup().